--- addnode.orig	2019-02-18 18:32:43.024088264 +0300
+++ addnode	2019-02-18 18:33:03.736258440 +0300
@@ -159,8 +159,7 @@
     configfile.chmod( 0640)
     configfile.chown( 0, user.gid) unless user.nil?
   rescue => e
-    puts "ERROR: can't set owner and/or permissions on `#{Conf.file}' (#{e.class}: #{e.message}).".red
-    return false
+    puts "WARNING: can't set owner and/or permissions on `#{Conf.file}' (#{e.class}: #{e.message}).".yellow
   end
 
   true
